<?xml version="1.0" ?>
<robot name="m0609">

  <material name="dr_gray">
    <color rgba="0.490 0.490 0.486 1.0"/>
  </material>
  <material name="dr_white">
    <color rgba="0.941 0.941 0.941 1.0"/>
  </material>
  
  <link name="base_link">
	<inertial>
	  <mass value="3.722"/>
	  <origin xyz="-1.2e-05 -6e-05 0.050723"/>
      <inertia ixx="0.011" ixy="-7.19e-06" ixz="-1.94e-06" iyy="0.011" iyz="4.9e-06" izz="0.014"/>
    </inertial>
	<visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_0_0.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_gray"/>
    </visual>
  </link>

  <joint name="joint1" type="revolute">
    <parent link="base_link"/>
    <child link="link1"/>
    <origin rpy="0 0 0" xyz="0 0 0.1525"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-6.28319" upper="6.28319" velocity="2.61799"/>
  </joint>

  <link name="link1">
	<inertial>
	  <mass value="7.446"/>
	  <origin xyz="7e-05 0.044281 0.143327"/>
      <inertia ixx="0.039" ixy="2.1e-05" ixz="-3.61e-05" iyy="0.023" iyz="-0.003" izz="0.039"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_1_0.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_white"/>
    </visual>
  </link>
  
  <joint name="joint2" type="revolute">
    <parent link="link1"/>
    <child link="link2"/>
    <origin rpy="0 -1.571 -1.571" xyz="0 0.006 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-6.28319" upper="6.28319" velocity="2.61799"/>
  </joint>

  <link name="link2">
	<inertial>
	  <mass value="10.569"/>
	  <origin xyz="-1.6e-05 0.189372 0.418558"/>
      <inertia ixx="0.820" ixy="1.19e-05" ixz="-0.000263" iyy="0.82" iyz="-0.008" izz="0.031"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_2_0.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_white"/>
    </visual>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>     
		<mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_2_1.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_gray">
        <color rgba="0.490 0.490 0.486 1.0"/>
      </material>
	  </visual>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>     
		<mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_2_2.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_white"/>
    </visual>	
  </link>

  <joint name="joint3" type="revolute">
    <parent link="link2"/>
    <child link="link3"/>
    <origin rpy="0 0 1.571" xyz="0.411 0 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.61799" upper="2.61799" velocity="3.14159"/>
  </joint>
  
  <link name="link3">
	<inertial>
	  <mass value="3.614"/>
	  <origin xyz="6e-06 0.07917 0.781807"/>
      <inertia ixx="0.012" ixy="-4.38e-06" ixz="-1.2e-06" iyy="0.008" iyz="0.002" izz="0.011"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>     
		<mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_3_0.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_white"/>
    </visual>
  </link>

  <joint name="joint4" type="revolute">
    <parent link="link3"/>
    <child link="link4"/>
    <origin rpy="1.571 0 0" xyz="0 -0.368 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-6.28319" upper="6.28319" velocity="3.92699"/>
  </joint>
  
  <link name="link4">
	<inertial>
	  <mass value="3.785"/>
	  <origin xyz="-2.1e-05 0.12337 1.148"/>
      <inertia ixx="0.177" ixy="-2.34e-06" ixz="-4.92e-06" iyy="0.161" iyz="-0.05" izz="0.022"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>     
		<mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_4_0.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_white"/>
    </visual>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>     
		<mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_4_1.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_white">
        <color rgba="0.941 0.941 0.941 1.0"/>
      </material>
    </visual>
  </link>

  <joint name="joint5" type="revolute">
    <parent link="link4"/>
    <child link="link5"/>
    <origin rpy="-1.571 0 0" xyz="0 0 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-6.28319" upper="6.28319" velocity="3.92699"/>
  </joint>
  
  <link name="link5">
	<inertial>
	  <mass value="2.598"/>
	  <origin xyz="-1.7e-05 0.06662 1.33103"/>
      <inertia ixx="0.007" ixy="-4.93e-06" ixz="-5.57e-06" iyy="0.004" iyz="-6.64e-05" izz="0.007"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>     
		<mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_5_0.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_white"/>
    </visual>
  </link>

  <joint name="joint6" type="revolute">
    <parent link="link5"/>
    <child link="link6"/>
    <origin rpy="1.571 0 0" xyz="0 -0.121 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-6.28319" upper="6.28319" velocity="3.92699"/>
  </joint>
  
  <link name="link6">
	<inertial>
	  <mass value="1.128"/>
	  <origin xyz="-0.000174 0.034708 1.40022"/>
      <inertia ixx="0.001" ixy="-2.79e-06" ixz="5.78e-06" iyy="0.001" iyz="-6.67e-06" izz="0.001"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>     
		<mesh filename="PATH/TO/dsr_description/meshes/m0609_white/MF0609_6_0.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="dr_gray"/>
    </visual>
  </link>
  
</robot>
